via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -382,24 +428,36 @@ export default function VideoPlayerApp({
               'flex flex-col'
             )}
           >
-            <Transcript
-              autoScroll={autoScroll}
-              transcript={transcript}
-              controlsRef={transcriptControls}
-              currentTime={timestamp}
-              filter={trimmedFilter}
-              onSelectSegment={segment => setTimestamp(segment.start)}
-            >
-              <div
-                data-testid="bucket-bar-channel"
-                className={classnames(
-                  // Provide a backdrop for bucket bar buttons. 20px of this
-                  // is overlaid by the sidebar's semi-transparent bucket
-                  // channel.
-                  'bg-gradient-to-r from-white to-grey-1 border-l w-[40px]'
-                )}
-              />
-            </Transcript>
+            {isLoading && (
+              <div className="flex justify-center p-8">
+                <Spinner data-testid="transcript-loading-spinner" size="md" />
+              </div>
```

Comment:
This spinner vis-a-vis the height of the transcript column/region needs a little attention, but needs to be done in context with https://github.com/orgs/hypothesis/projects/118/views/1?pane=issue&itemId=31604133

I can pick this up on my next pass.